api4/user.go

```diff
@@ -599,6 +600,21 @@ func getFilteredUsersStats(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(stats.ToJson()))
 }
 
+func getGeneralUserStats(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
I think exposing this is not strictly needed for the functionality, I would say here we should expose the "remaining available slots for new users".

Reply:
Oh yeah! I think that's much better. Thanks a lot